SPECS/crash/crash.spec

```diff
@@ -9,6 +9,7 @@ URL:           https://github.com/crash-utility/crash
 # Source0:     https://github.com/crash-utility/%{name}/archive/%{version}.tar.gz
 Source0:       %{name}-%{version}.tar.gz
 Source1:       https://ftp.gnu.org/gnu/gdb/gdb-7.6.tar.gz
+Patch0:        crash-printk-fix.patch
```

Comment:
If we apply these changes to crash, will this updated crash still work on systems that are still on 5.4?
If this needs to be coupled with 5.10, we may need to enforce a dependency on the kernel 5.10 version packages

Reply:
Yes, these changes are backwards compatible with 5.4 (I tested locally). The patches simply add additional support for the newer printk functionality